src/Voice.tsx

```diff
@@ -688,6 +688,11 @@ export class Voice extends EventEmitter {
       case 'ios':
         await settleNativePromise(NativeModule.voice_showNativeAvRoutePicker());
         return;
+      case 'android':
```

Comment:
This used to be a no-op on Android before the Expo changes. As part of Expo, I removed the no-op code in the native layer, and now we no-op on the JS layer with this change.